src/components/login/login_form.js

```diff
@@ -1,34 +1,59 @@
 import React, {useState} from "react"
-//import axios from "axios"
+import axios from "axios"
 
 const base = {
-    emailID: "login-email",
+    emailId: "login-email",
     passwordId: "login-password",
-    formId: "login-form",
+    formClass: "login-form",
     submitId: "login-submit"
 }
 
-function LoginForm () {
+function LoginForm (props) {
+
     let [email, setEmail] = useState("")
     let [password, setPassword] = useState("")
 
     const handleChange = (event) => {
-            if(event.target.id === base.emailID ) setEmail(email = event.target.value)
+            if(event.target.id === base.emailId ) setEmail(email = event.target.value)
             if(event.target.id === base.passwordId ) setPassword(password = event.target.value)
-        }   
+        }  
+
+    const clearFields = (arrFields = []) => {
+        arrFields.forEach(field => document.getElementById(field).value ="")
+    }
     
-    const handleSubmit = (event) => {
-         event.preventDefault()
-         
-    }    
+    const clearState = () => {
+        setEmail(email ="")
+        setPassword(password ="")
+    }
+    
+    const handleSubmit = async (event) =>  {
+        event.preventDefault()
+        try{
+            const token = await axios({
+                 method:"POST",
+                 url: "http://127.0.0.1:4000/lender/login",
+                 data:{
+                     email,
+                     password
+                 }
+             })
+             console.log(token)
+        }catch(error) {
+            props.handleErrorLogin(error.response.data)
+            clearFields([base.emailId, base.passwordId])
+            clearState()
+            document.getElementById(base.emailId).focus()
```

Comment:
For now, it's ok to have it like this. I recommend you to read this section of the React documentation to handle this type of behavior [Refs and the DOM](https://reactjs.org/docs/refs-and-the-dom.html)

Reply:
ok, I will reed the documentation and implement the changes in the next commit.